01/OleksiyPol/app.js

```diff
@@ -0,0 +1,95 @@
+const chalk = require('chalk')
+const yargs = require('yargs')
+const fs = require("fs")
+
+
+function readNotes(){
+  let rawNotes = fs.readFileSync('notes.json')
+  let notes = JSON.parse(rawNotes)
+  return notes
+}
+function writeNotes(notes){
+  let data = JSON.stringify(notes,null,2)
+  fs.writeFileSync('notes.json', data)
+}
+
+
+yargs.command({
+  command:'add',
+  describe:'Add new note',
+  builder:{
+    title:{
+      describe:"Note title",
+      demandOption: true,
+      type:"string"
+    },
+    body:{
+      describe:"Note body",
+      demandOption: true,
+      type:"string"
+    }
+  },
+  handler: function(argv){
+    let newNote = {'title':argv.title, 'body':argv.body}
```

Comment:
The better way will be to create an own function called, for example, createNewNote and move It into the notes.js file.